Feature request template (#4)
Libra-foundation/.github: ISSUE_TEMPLATE/NEW_FEATURE_TEMPLATE.md

---
name: Feature request \
about: Suggest an idea for this project \
title: Feature request \
labels: feature

---

### Summary

> One paragraph explanation of the feature.

Reviewer: ```suggestion
Explane of the feature you would like. You may add images, exemples, anything you think is relevant to your point.
```

### Motivation

> Why is this feature of general interest?

Reviewer: ```suggestion
Why is this feature of general interest ? Tell us why you want this feature, in what it is a good feature to add.
```

### Implementation

> What solution do you suggest?

Reviewer: ```suggestion
What solution do you suggest ? You can add here technical details if you have the knowledge or just describe the general solution you prpose (ie: add a button, a macro ...).
```

### Formatting

> Please use markdown in your issue message. A useful summary of commands can be found [here](https://guides.github.com/pdfs/markdown-cheatsheet-online.pdf).

Reviewer: I like the idea of integrating a markdown guide to the templates, I think we should downstream this to our others templates. I however dislike how it is placed, maybe adding it as a disclaimer-ish thing on top of the issue ?

Author: Yes, I agree. I reopen ISSUE_TEMPLATE branch to add it in template ?

I think we will add it inside prerequisites part.

Reviewer: I do not think this is needed, you can simply apply the change in this branch. There should not be any problem.

---

name: Feature request \
about: Suggest an idea for this project \
title: Feature request \
labels: feature

---

> 
> ### Prerequisites
> 

Reviewer: ```suggestion
```

> ---

Reviewer: Why did you remove the header ? Headers are providing helpful metadata about the template.

> Please use markdown in your issue message. A useful summary of commands can be

Reviewer: ```suggestion
Issues are formated using markdown, a useful summary of the features provided by github can be
```

found [here](https://guides.github.com/pdfs/markdown-cheatsheet-online.pdf).

### Summary

> Explain of the feature you would like. You may add images, exemples, anything you think is relevant to your point.

Reviewer: ```suggestion
Explain the feature you would like. You may add images, exemples, anything you think is relevant to your point.
```
